test-kitchen.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "winrm",              "~> 2.0"
   gem.add_dependency "winrm-elevated",     "~> 1.0"
   gem.add_dependency "winrm-fs",           "~> 1.1"
+  gem.add_dependency "chef-utils",         ">= 16.4.35"
```

Comment:
This version has a patch in it which supports windows PATHEXT in the which helper